webui/react/src/utils/fonts.ts

```diff
@@ -0,0 +1,50 @@
+import { ValueOf } from 'shared/types';
+
+export const ThemeFont = {
+  Code: 'var(--theme-font-family-code)',
+  UI: 'var(--theme-font-family)',
+} as const;
+
+export const Header = {
```

Comment:
Are these actually used? It looks like you defined a separate enum in the components themselves.

Reply:
Where would we use these other than the header? And if line height is something we want then why _doesn't_ header take them? (not clear to me that we do want that, was it a request from design?)

Reply:
In that case what you have here doesn't accomplish what Alex asked for because it's not actually usable.

Reply:
By CSS variables do you mean the `ParagraphSizes` you defined?

Reply:
How do you use that with `Paragraph` or `Header`?

Reply:
I can read the code. You just said that the enums in the ts code aren't the same as the css vars.